packages/manager/apps/pci-kubernetes/src/pages/new/New.page.tsx

```diff
@@ -266,24 +270,26 @@ export default function NewPage() {
             step={stepper.nodeType.step}
           />
         </StepComponent>
+        {nodePoolEnabled && (
+          <StepComponent
+            order={6}
+            {...stepper.nodeSize.step}
+            title={tListing('kube_common_node_pool_autoscaling_title')}
+            edit={{
+              action: stepper.nodeSize.edit,
+              label: tStepper('common_stepper_modify_this_step'),
+              isDisabled: isCreationPending,
+            }}
+          >
+            <NodeSizeStep
+              isMonthlyBilling={stepper.form.isMonthlyBilled}
+              onSubmit={stepper.nodeSize.submit}
+              step={stepper.nodeSize.step}
+            />
+          </StepComponent>
+        )}
         <StepComponent
-          order={6}
-          {...stepper.nodeSize.step}
-          title={tListing('kube_common_node_pool_autoscaling_title')}
-          edit={{
-            action: stepper.nodeSize.edit,
-            label: tStepper('common_stepper_modify_this_step'),
-            isDisabled: isCreationPending,
-          }}
-        >
-          <NodeSizeStep
-            isMonthlyBilling={stepper.form.isMonthlyBilled}
-            onSubmit={stepper.nodeSize.submit}
-            step={stepper.nodeSize.step}
-          />
-        </StepComponent>
-        <StepComponent
-          order={7}
+          order={nodePoolEnabled ? 7 : 6}
```

Comment:
What is 7 or 6 ? We can add constant to readability ?

```suggestion
          order={nodePoolEnabled ? 7 : 6}
```